MapboxCoreNavigation/RouteController.swift

```diff
@@ -424,19 +424,23 @@ extension RouteController: CLLocationManagerDelegate {
 
     func updateRouteLegProgress(for location: CLLocation) {
         let currentDestination = self.routeProgress.currentLeg.destination
-        guard let remainingVoiceInstructions = routeProgress.currentLegProgress.currentStepProgress.remainingSpokenInstructions else { return }
```

Comment:
This guard is what was throwing us off. We'd unconditionally return without calling `didArriveAt:` if not using spoken instructions.